### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
Screenshot (1482)
Screenshot (1481)

Features added: completed page template and did avatar component library

### CODE REVIEW

File: Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css");
+:root {
+    --vodka-UI-background-color: white;
```
Comment (bharati-21) [inline Avatar/avatar.css:4]: If these are colors common to the component library, you can add them to a separate shared file called `variables.css`. If not, then this looks good!

File: Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+footer {
+    width: 80%;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 1.6rem;
+    margin-left: 1.2rem;
+    margin-right: 1.2rem;
+}
+
+footer a {
```
Comment (bharati-21) [inline Avatar/avatar.css:155]: I see the styles for a being repeated twice above. Maybe combine them to form a class? Say, nav link or link?

File: Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    opacity: 1;
+}
+
+iframe {
+    margin-top: 2rem;
+    width: 97%;
+}
+
+
+/* profile avatar */
```
Comment (bharati-21) [inline Avatar/avatar.css:172]: It makes sense for me to add avatar styles separately from the component styles. And reuse the styles by importing on this file.

File: Avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
```
Comment (bharati-21) [inline Avatar/avatar.html:5]: As suggested in the PR review session, you could add an meta author tag.

File: Avatar/avatar.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                <p class="content-paragraph">
+                    Avatars are images that users can set as their profile picture. On GitHub, they're always going to be rounded squares. They can be custom photos, uploaded by users, or generated as Identicons as a placeholder.
+                </p>
+                <h1 class="content-heading">Profile Avatar</h1>
+                <p class="content-paragraph">
+                    Profile avatars can be created by adding class
+                    <span class="highlight-word">avatar</span> in img tag.
+                </p>
+                <div class="example-div">
+                    <img class="avatar" src="https://images.unsplash.com/photo-1593085512500-5d55148d6f0d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&au" alt="profile avatar" />
```
Comment (bharati-21) [inline Avatar/avatar.html:49]: Would suggest using images after adding them to your assets or images/ folder in your project.

File: Avatar/avatar.html [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+                    <img src="https://www.absolut.com/globalassets/images/products/absolut-citron/absolut-citron_1x1.jpg" class="avatar" alt="profile avatar" />
+                    <img src="https://www.ciroc.com/images/en-row/recipes/blue-stone-champagne-cosmo-bottle-drinks.webp" class="avatar" alt="profile pic" />
+                </div>
+                <code class="carbon-code">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522url%2520of%2520image%2522%2520alt%253D%2522profile%2520avatar%2522%2520%252F%253E"
+              style="
```
Comment (bharati-21) [inline Avatar/avatar.html:56]: Kindly avoid inline styles.

File: Avatar/avatar.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const scrollbtn = document.querySelector(".direct-upward");
+console.log(scrollbtn);
```
Comment (bharati-21) [inline Avatar/avatar.js:2]: In case you forget... please do remove the console.log() statement. 👍🏽

**Review by bharati-21 — APPROVED** [review]
Wonderful use of colors and your site looks very neat, clean, and beautiful! ⭐ 

I did leave a few suggestions that you could follow. But apart from these, everything looks good to me.

**Review by SukanyaSen530 — APPROVED** [review]
Look's good, can be merged :)

**Pull request merged**